### DISCUSSION TITLE
How to use navbar ?

### DISCUSSION THREAD

**Discussion by jwojda** [tailwindlabs/tailwindcss#4116] [Help]
Hello, 

i tried to use navbar [https://tailwindui.com/components/application-ui/navigation/navbars](https://tailwindui.com/components/application-ui/navigation/navbars) but in Mobile : the menu button doesn't work : Menu is always open.

I used : 
```
<link href="tailwind.css" rel="stylesheet">
<link href="css/style.css" rel="stylesheet">
<script type="text/javascript" src="js/tailwind.config.js"></script>

```
Something else?

**Comment by MartinHughes-BPC** [comment]
You'll need to write your own code to do this, or you could use the React or Vue example on TailwindUI if you use either of these frameworks.

Cheers,
Martin.
